### PR TITLE
Fix MyProfile

### PR DESCRIPTION
**Pull request by lapaset**
back: 
- Changed validation so that it requires current password only when updating password
front:
- Separated Update Information and Update Password forms
- Added separate alerts
- Refactored the component to a MyProfile folder

### CODE REVIEW

File: frontend/src/components/user/MyProfile/MyProfile.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+            onClick={userProfileModal.handleClickOpen}
+          >
+            User Profile
+          </Button>
+          <p>(this button will be removed)</p>
+        </Grid>
+        <Grid item xs={12} sm={7} className={classes.column}>
+          <UpdateInformation {...updateInformationProps} />
+          <UpdatePassword {...updatePasswordProps} />
+        </Grid>
```
Comment (T7Q) [inline frontend/src/components/user/MyProfile/MyProfile.js]: nice!

File: frontend/src/components/user/MyProfile/MyProfile.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
+import React, { useState, useEffect } from 'react';
```
Comment (T7Q) [inline frontend/src/components/user/MyProfile/MyProfile.js]: This is the start of MyProfile, it's better to name this file Index.js for better navigation in files

**Review by T7Q — CHANGES_REQUESTED** [review]

File: frontend/src/components/user/MyProfile/MyProfile.js (hunk 2 above)
Comment (lapaset) [reply frontend/src/components/user/MyProfile/MyProfile.js]: Done :)

**Comment by iljaSL** [comment]
I guess it's ready to go after the merge conflict is solved.

**Pull request merged**